josebasilioo/xpace-admin
# feature(develop): added new model and route

## What i did 
- created a new model to header administration
- created a new route to header database

## How to test
1. You need to clone the repository and install all the dependencies
2. To test you need to run the server with 'node .'
3. Go to localhost:3001/admin and make changes on the Header section
4. To verify the database changes go to localhost:3001/api/header
